belt-mac/Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["cryptography", "no-std"]
 
 [dependencies]
 belt-block = { version = "0.1" }
-digest = { version = "0.10.3", features = ["mac"] }
+digest = { version = "0.10.7", features = ["mac"] }
```

Comment:
Ugh, probably shouldn't have merged this as-is, it bumps all of these patch versions